tests/unit/components/DocumentationTopic/DocumentationHero.spec.js

```diff
@@ -77,6 +78,16 @@ describe('DocumentationHero', () => {
     expect(wrapper.find('.short-hero').exists()).toBe(false);
   });
 
+  it('renders the right classes based on `shouldShowLanguageSwitcher` prop', () => {
+    const wrapper = createWrapper();
+    expect(wrapper.find('.extra-bottom-padding').exists()).toBe(true);
+
+    wrapper.setProps({
+      shouldShowLanguageSwitcher: false,
+    });
+    expect(wrapper.find('.extra-bottom-padding').exists()).toBe(false);
```

Comment:
You can also so something like this

```suggestion
    const content = wrapper.find('.documentation-hero__content');
    
    expect(content.classes()).toContain('extra-bottom-padding');

    wrapper.setProps({
      shouldShowLanguageSwitcher: false,
    });
    expect(content.classes()).not.toContain('extra-bottom-padding');
```